src/main/java/org/b612foundation/adam/datamodel/AdamObject.java

```diff
@@ -0,0 +1,48 @@
+package org.b612foundation.adam.datamodel;
+
+import java.util.Objects;
+
+public class AdamObject {
+
+  /** Unique id for an object, generated on creation. */
+  private String uuid;
+
+  /** Project to which this object belongs. */
```

Comment:
I would move this to metadata. At this level we don't even know what "project" means.

Reply:
Valid point. Moved.